src/index/run.go

```diff
@@ -83,6 +83,8 @@ func (idx Index) Run(
 	}
 	if doRun && hasCommand {
 		logger.Info.Printf("running stage %s\n", stagePath)
+		logger.Info.Printf("  command    : %s\n", stg.Command)
```

Comment:
How about putting this in the debug log and combining them into one print statement? Something like:

``` go
logger.Debug.Printf("(%s) %s\n", std.WorkingDir, std.Command)
```